google/cloud/ndb/context.py

```diff
@@ -544,6 +546,12 @@ def policy(key):
 
     set_memcache_timeout_policy = set_global_cache_timeout_policy
 
+    def set_retry_state(self, state):
+        self.retry = state
```

Comment:
This instance variable is only accessed through setters and getters, so let's indicate it's private, like `self._retry`